src/databricks_clean_room_orchestrator/client.py

```diff
@@ -273,6 +273,10 @@ def _prepareAndRunNotebookHelper(
       if (response["workspace_status"] != "PROVISIONING"):
         raise RuntimeError("Workspace could not be provisioned")
       time.sleep(10)
+    
+    # Finalize station workspace setup to update the workspace detail.
+    print("Finalize station workspace setup")
+    self._rest_client.setupStationResource(self._clean_room, self._station_name, Resource.WORKSPACE)
```

Comment:
Would have been better if we had a separate resource / step for this... let's make it right after as this is a hack. cc @connorlien you would have to incorporate / fix this logic in your BYOT integration.